### PR TITLE
feat(t13): added maps ui

### PR DESCRIPTION
**Pull request by AlekseySamokhin**
I installed docker and pgadmin, set up a container on the backend, and added a few rows to the map table locally. on the front i got all the data and added pagination using offset and limit.

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ import '../styles/globals.css';
 const clientSideEmotionCache = createEmotionCache();
 
 const client = new ApolloClient({
-  uri: 'https://rickandmortyapi.com/graphql',
+  uri: 'http://localhost:4000/graphql',
```
Comment (killmenot) [inline pages/_app.tsx:19]: Let's use env var here

File: pages/maps/index.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
+import { useState } from 'react';
+import { useQuery, gql } from '@apollo/client';
+
+import { DataList } from '@/components/common';
+import { Container } from '@mui/system';
+import { Pagination } from '@/components/common/Pagination/Pagination';
+
+const QUERY_ALL_MAPS = gql`
```
Comment (killmenot) [inline pages/maps/index.tsx:8]: Let's do not keep query in page. Think about separation. Also please think about how we can use strong typing for queries

File: pages/maps/index.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
+import { useState } from 'react';
+import { useQuery, gql } from '@apollo/client';
+
+import { DataList } from '@/components/common';
+import { Container } from '@mui/system';
```
Comment (killmenot) [inline pages/maps/index.tsx:5]: probably Container should be part of layout not page, no?

File: pages/maps/index.tsx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  },
+  {
+    id: 4,
+    label: 'Author',
+    render: (x: any) => x.author,
+  },
+];
+
+const Maps: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
```
Comment (killmenot) [inline pages/maps/index.tsx:47]: merge these states to a single object and use `immer` to update the state

File: pages/maps/index.tsx (hunk 2 above)
Comment (AlekseySamokhin) [reply pages/maps/index.tsx:8]: I will study the issue of typing gql queries.

**Review by killmenot — APPROVED** [review]

**Pull request merged**